cloud/shared/bin/lib/terraform.sh

```diff
@@ -18,24 +13,22 @@ readonly TERRAFORM_APPLY="${TERRAFORM_BASE_COMMAND} \
 #######################################
 function terraform::perform_apply() {
   if [[ "${CIVIFORM_MODE}" == "dev" ]]; then
-    "$(${TERRAFORM_BASE_COMMAND} init -upgrade)"
+    terraform "${TERRAFORM_DIR}" init -upgrade
```

Comment:
this seems not quite right?

Reply:
Thanks for catching this!  Yes, these are leftover from my first attempt